lua/wire/sh_wirenet.lua

```diff
@@ -0,0 +1,210 @@
+--[[
+	Consolidated networking library for Wiremod.
+	Trivial Networking - A wrapper for sending very small net messages that don't need a whole networkstring.
+]]
+
+local Net = WireLib.Net or {}
+WireLib.Net = Net
+
+-- Trivial Networking
+local SIZE = 10 -- 1023 names
```

Comment:
Nitpick but I'd prefer `MAX_NAMES` and then just `local SIZE = math.ceil(math.log(MAX_NAMES + 1, 2))`

Reply:
That's gonna bother me if someone accidentally puts the wrong `MAX_NAMES` so it makes size 1 bit more than it needs to be.